docker-compose.yml

```diff
@@ -5,4 +5,6 @@ services:
       context: .
     volumes:
       - .:/code
-    command: tail -f /dev/null
\ No newline at end of file
+    command: tail -f /dev/null
+    # TODO: load this from an environment variable so M1 macs can easily use this
+    platform: linux/x86_64
```

Comment:
I need to figure out the best way to do this. Without it, M1 macs will see `/lib64/ld-linux-x86-64.so.2: No such file or directory`